eth/eventdatahandler/task_executor_test.go

```diff
@@ -45,10 +46,99 @@ func TestExecuteTask(t *testing.T) {
 	}
 	task := NewStartValidatorTask(edh.taskExecutor, share)
 	require.NoError(t, task.Execute())
+	if logs.Len() == 0 {
+		t.Errorf("No logs")
+	} else {
+		entry := logs.All()[len(logs.All())-1]
+		require.Equal(t, "validator wasn't started", entry.Message)
```

Comment:
Let's also think how to test the happy flow

Reply:
Not sure here what would be the happy flow - we need to set proper env vars probably to have an event with the valid sig

Reply:
We need to add what's missing to make it start, e.g. share metadata